src/gennylib/benchmark/orchestrator_benchmark.cpp

```diff
@@ -32,7 +30,6 @@ using namespace std::chrono;
 TEST_CASE("Orchestrator Perf", "[benchmark]") {
     const auto dur = milliseconds{200};
 
-    genny::metrics::Registry metrics;
```

Comment:
Due to https://bugs.archlinux.org/task/61786, I wasn't able to run `perf` on Arch Linux to profile why after the changes from 5a8c92e7e1003af6ed099419a6a2fe88f8ca273e slowed things down. There's likely some difference in the generated code that may be impacting inlining of some of the function calls. What's especially odd to me is that `genny::Orchestrator` doesn't even make use of `genny::metrics::Registry`.